### PR TITLE
Adding Will Pirkle Korg35 VA low pass filter

### PR DESCRIPTION
**Pull request by owenvallis**

### CODE REVIEW

**Comment by owenvallis** [comment]
Just updated the filter base class to use doubles

File: WPKorg35/WPKorg35.cpp [hunk 1]
```diff
@@ -0,0 +1,298 @@
...
+    // for UGen's only: add tick function
+    QUERY->add_ugen_func(QUERY, wpkorg35_tick, NULL, 1, 1);
+    
+    // NOTE: if this is to be a UGen with more than 1 channel, 
+    // e.g., a multichannel UGen -- will need to use add_ugen_funcf()
+    // and declare a tickf function using CK_DLL_TICKF
+
+    // example of adding setter method
+    QUERY->add_mfun(QUERY, wpkorg35_setResonance, "float", "resonance");
+    // example of adding argument to the above method
```
Comment (ericheep) [inline WPKorg35/WPKorg35.cpp]: I'd recommend adding some documentation to the class and functions here, something like this.

`QUERY->doc_func(QUERY, "Set filter resonance. ");`

ABSaturator, PowerADSR, and WinFuncEnv have further examples of how to do this.

Comment (owenvallis) [inline WPKorg35/WPKorg35.cpp]: I added descriptions at the top of the class. I'll move them to use the doc_func

Comment (ericheep) [inline WPKorg35/WPKorg35.cpp]: Cool, I'd add some documentation to the general README too.

**Pull request merged**